src/shared/previewUtils.ts

```diff
@@ -495,4 +508,18 @@ export class PreviewUtils {
     logger?.debug(`Extracting archive ${zipFilePath}`);
     await CommonUtils.executeCommandAsync(cmd, logger);
   }
+
+  private static async getEntityId(username: string): Promise<string> {
+    const identityData = await ConfigUtils.getIdentityData();
+    let entityId: string | undefined;
+    if (!identityData) {
+      throw new Error(messages.getMessage('error.identitydata'));
```

Comment:
Since this is an `async` function with a `Promise` return type, instead of throwing I would recommend returning `Promise.reject(new Error(...))`